db/schema.rb

```diff
@@ -126,16 +131,6 @@
     t.integer "business_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.string "sundayopen"
```

Comment:
I'll look into how to sort this out, migration wise. I left the existing "Monday, Tuesday, etc..." columns in the schema, and the controller is permitting the open/close columns. I'll add the previous day `:monday` params back to the controller in addition to the new stuff.

Reply:
I fixed this is my pull request